lang/uYouPlus.bundle/ko.lproj/Localizable.strings

```diff
@@ -13,10 +15,10 @@
 "HIDE_PREVIOUS_AND_NEXT_BUTTON_DESC" = "비디오 재생 오버레이에서 이전 재생, 다음 재생 버튼을 숨깁니다.";
 
 "YT_MINIPLAYER" = "모든 YouTube 영상에서 미니 플레이어 활성화";
-"YT_MINIPLAYER_DESC" = "아동용 영상일 경우.";
+"YT_MINIPLAYER_DESC" = "유튜브 키즈의 영상 같은 경우.";
 
 "CAST_CONFIRM" = "TV 전송 알림 (YTCastConfirm)";
-"CAST_CONFIRM_DESC" = "의도치 않게 TV로 영상을 전송하지 않도록 보내기 전에 확인 메시지를 표시합니다.";
+"CAST_CONFIRM_DESC" = "의도하지 않게 TV로 영상을 전송하지 않도록 보내기 전에 확인 메시지를 표시합니다.";
```

Comment:
I think the earlier one is better than this (just personal thoughts)

Reply:
Thanks for the feedback. Edited as you advised.